app/src/androidTest/java/de/test/antennapod/handler/FeedParserTestBase.java

```diff
@@ -38,7 +38,8 @@
 
     @Before
     public void setUp() throws Exception {
-        Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
+        Context context = InstrumentationRegistry
+                .getInstrumentation().getTargetContext();
```

Comment:
I think the line break is not needed here. Checkstyle should be happy without the line break and I think it makes the code harder to read. The same applies to other changes in this commit.